features/liquidity/components/UnderlyingAssetRow.tsx

```diff
@@ -48,8 +49,8 @@ export const UnderlyingAssetRow = ({
         />
         <Text variant="link">{tokenSymbol}</Text>
       </Inline>
-      <Inline align="center" gap={4}>
-        <Inline gap={6}>
+      <Inline align="center" gap={2}>
+        <Inline gap={2} css={{ alignItems: 'baseline' }}>
```

Comment:
Cool looks like we might want to add 'baseline' as another value you would pass to the `align` prop

Reply:
You mean why have align and justify content as variants rather than just pass css object directly? Well multiple reasons. One is because each render javascript will create a new instance of your styles object and second is it adds unnecessary clutter